src/issuance/mdoc.rs

```diff
@@ -28,6 +28,7 @@ pub type Namespaces = BTreeMap<String, BTreeMap<String, ciborium::Value>>;
 pub struct Mdoc {
     pub doc_type: String,
     pub mso: Mso,
+    #[serde(rename = "nameSpaces")]
     pub namespaces: IssuerNamespaces,
```

Comment:
This isn't a standard data structure, it's just what is used by this library to represent an issued mDoc, so there isn't a "correct" field name.